patches/api/0005-Adventure.patch

```diff
@@ -2471,15 +2471,23 @@ index 3d84a75a6999129353a9ead48f9db25bc80f685b..a5010cb46aa5ef69c559ef9fc717f903
       * @param url The URL from which the client will download the resource
       *     pack. The string must contain only US-ASCII characters and should
       *     be encoded as per RFC 1738.
-@@ -1358,6 +1570,7 @@ public interface Player extends HumanEntity, Conversable, OfflinePlayer, PluginM
+@@ -1327,6 +1539,7 @@ public interface Player extends HumanEntity, Conversable, OfflinePlayer, PluginM
+      * @throws IllegalArgumentException Thrown if the hash is not 20 bytes
+      *     long.
+      */
++    @Deprecated // Paper
+     public void setResourcePack(@NotNull String url, @Nullable byte[] hash);
```

Comment:
Can you also fix the faulty javadocs of the old setResourcePack methods that do not FQN the Component ?

Given we don't import them, the javadocs link to the wrong method rn.

Reply:
Yes, sure